blockstore/splitstore/markset_badger.go

```diff
@@ -188,6 +136,140 @@ func (s *BadgerMarkSet) Has(c cid.Cid) (bool, error) {
 	}
 }
 
+func (s *BadgerMarkSet) Visit(c cid.Cid) (bool, error) {
+	key := c.Hash()
+	pendKey := string(key)
+
+	checkPending := func() (bool, error) {
```

Comment:
the behavior here is flipped. I'd expect `checkPending` to return _true_ if a write is pending.

Reply:
yeah, the name is perhaps unfortunate; the return signature is (visit, err) -- perhaps I should just name the return variables to make it clear what it returns.

Reply:
I guess I could rename it to `visitPending` so that it is consistent with its return signature.

Reply:
made it a method called `tryPending` and reversed the signature to return `(has, error)`.